conf/messages.en

```diff
@@ -332,16 +332,24 @@ confirmation.print.afterLink = to make a record of your submission.
 confirmation.mayNeedToDo.h2 = What you may need to do
-confirmation.mayNeedToDo.p1 = This explanation for a shortage or excess has been submitted to HMRC only. If you are a Northern Ireland warehousekeeper receiving goods from an EU member state, this information will also be visible to the state of dispatch.
+confirmation.mayNeedToDo.p1.dutypaid = This explanation for a shortage or excess has been submitted to HMRC only. If you are a Northern Ireland certified or temporary certified consignee receiving goods from an EU member state, this information will also be visible to the authorities in the state of dispatch.
+confirmation.mayNeedToDo.p1.dutysuspended = This explanation for a shortage or excess has been submitted to HMRC only. If you are a Northern Ireland warehousekeeper receiving goods from an EU member state, the information will also be visible to the authorities in the state of dispatch.
 confirmation.mayNeedToDo.shortage.h3 = If you recorded a shortage
-confirmation.mayNeedToDo.shortage.p1 = A duty point may be created for any goods in this movement that are short. This means HMRC will require duty to be paid on those goods by the person or organisation that is holding them.
+confirmation.mayNeedToDo.shortage.p1.dutypaid = You may be able to reclaim overpaid duty on the goods in this movement that are short.
+confirmation.mayNeedToDo.shortage.p2.dutypaid = For information on what to do next if you discover shortages, see
+confirmation.mayNeedToDo.shortage.p2.dutypaid.urltext = [Section 6.5 of Excise Notice 204b].
+confirmation.mayNeedToDo.shortage.p1.dutysuspended = A duty point may be created for any goods in this movement that are short. This means HMRC will require duty to be paid on those goods by the person or organisation that is holding them.
 confirmation.mayNeedToDo.excess.h3 = If you recorded an excess
-confirmation.mayNeedToDo.excess.p1 = When the excess goods match what’s on the electronic administrative document (eAD) you must:
-confirmation.mayNeedToDo.excess.bullet1 = keep a full audit trail of the onward movement of goods you do not keep
-confirmation.mayNeedToDo.excess.bullet2 = accept the duty liability for any goods you keep
-confirmation.mayNeedToDo.excess.p2 = When the excess goods do not match what’s on the eAD:
-confirmation.mayNeedToDo.excess.bullet3 = a duty point will be created for those goods
-confirmation.mayNeedToDo.excess.bullet4 = the person or organisation holding the goods must immediately pay the duty on them
+confirmation.mayNeedToDo.excess.p1.dutypaid = You must account for the UK duty on all excesses. Notify the duty-paid movements team immediately and they’ll give you advice on how to proceed.
+confirmation.mayNeedToDo.excess.p2.dutypaid = For information on what to do next if you discover excesses, see
+confirmation.mayNeedToDo.excess.p2.dutypaid.urltext = [Section 6.6 of Excise Notice 204b].
```

Comment:
Don't think this should have the square brackets?

Reply:
Pushed a change to remove the square brackets